azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/fragment/calling/participantlist/ParticipantListView.kt

```diff
@@ -48,9 +48,8 @@ internal class ParticipantListView(
         viewLifecycleOwner.lifecycleScope.launch {
             avatarViewManager.getRemoteParticipantsPersonaSharedFlow().collect {
                 if (participantListDrawer.isShowing) {
-                    updateRemoteParticipantListContent(
-                        viewModel.viewViewModelStateFlow.value.remoteParticipantList,
-                        viewModel.viewViewModelStateFlow.value.totalActiveParticipantCount
+                    updateParticipantListContent(
+                        viewModel.viewViewModelStateFlow.value,
```

Comment:
I would consider renaming these parameters on the next pass.

`viewModel.viewViewModelStateFlow` is kind of verbose. `viewViewModel` isn't really connected logically to participants in any naming sort of way, and the stateflow is redundant as java is type-safe so we know the types they don't need to be in the name.

I'd expect this to read something like
updateParticipantListContent(viewModel.participantListContent)